lib/bump/version_control.rb

```diff
@@ -0,0 +1,44 @@
+module Bump
+  class VersionControlNotFoundError < StandardError; end
+
+  class VersionControl
+    def self.git?
```

Comment:
use class << self + private to make only the methods public that you want to use from the outside